jsontext/encode.go

```diff
@@ -274,6 +274,7 @@ func (e *encoderState) UnwriteEmptyObjectMember(prevName *string) bool {
 	b = b[:len(b)-n]
 	b = jsonwire.TrimSuffixWhitespace(b)
 	b = jsonwire.TrimSuffixByte(b, ':')
+	b = jsonwire.TrimSuffixWhitespace(b)
```

Comment:
i think this is needed, but am not really sure about this section of code or why it worked previously without it even on multiline/expanded json.